operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/RetryConfiguration.java

```diff
@@ -1,26 +1,23 @@
 package io.javaoperatorsdk.operator.api.config;
 
-public interface RetryConfiguration {
```

Comment:
This is a non backwards compatible change, pls create a separate annotation, and mark the former interfaces deprecated